Tests/HtcMock/Server/src/SampleComputerService.cs

```diff
@@ -122,7 +122,7 @@ public override async Task<Output> Process(ITaskHandler taskHandler)
         await taskHandler.CreateTasksAsync(readyRequests.Select(r =>
                                                                 {
                                                                   var taskId = taskHandler.SessionId + "%" + r.Id;
-                                                                  logger_.LogDebug("Create task {taskId}",
+                                                                  logger_.LogDebug("Create task {task}",
```

Comment:
Reading on github is not easy sometimes... Where is the variable `task`?

Reply:
It is not a variable, it is a way to name a variable in logs so that it is searchable. The actual value is passed as another argument.